R/dataframe.R

```diff
@@ -182,24 +182,63 @@ as_data_frame.matrix <- function(x, ...) {
   as_data_frame(as.data.frame(x))
 }
 
-#' Convert row names to an explicit variable.
+#' Conversion between rownames and a column in data frame
+#'
+#' @details add_rownames: Convert row names to an explicit variable.
```

Comment:
I actually like the help page of ?BatchJobs::findNotDone and related functions: https://github.com/tudo-r/BatchJobs/blob/2fdea115f8b34dabf447247175bca0bcb891f9cb/R/findStatus.R#L28 -- this is similar to your style, but uses \code{}.